src/mono/wasm/debugger/DebuggerTestSuite/CallFunctionOnTests.cs

```diff
@@ -164,8 +158,7 @@ public async Task RunOnArrayReturnEmptyArray(string eval_fn, string bp_loc, int
                    await CheckProps(obj_own.Value["result"], new
                    {
                        length = TNumber(ret_len),
-                       // __proto__ returned by js
-                   }, $"obj_own", num_fields: is_js ? 2 : 1);
+                   }, $"obj_own", num_fields: 1);
```

Comment:
super-nit: You can drop the `num_fields` arg now, since we are passing only one member now (`length`), so it will expect the same number of fields.

Reply:
done.